astroplan/conftest.py

```diff
@@ -21,3 +21,14 @@
 # within the tests
 from .utils import _mock_remote_data
 _mock_remote_data()
+
+def pytest_configure(config):
+    try:
```

Comment:
Probably should also try importing `nose` here.
That's the matplotlib test runner, and if I remember correctly, there's a cryptic error message from pytest if matplotlib is there, but nose.